.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```

Comment:
Re: [signalapp/Signal-Android] Cannot create backup due to java.io.IOException "Renaming temporary backup file failed!" (#7692)